content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -14,11 +14,18 @@ is generally used by cluster components to validate the API server's
 certificate, by the API server to validate kubelet client certificates, etc. To
 support this, the CA certificate bundle is distributed to every node in the
 cluster and is distributed as a secret attached to default service accounts.
+
 Optionally, your workloads can use this CA to establish trust. Your application
```

Comment:
The CA distributed to pods in the service account token volume is *only* guaranteed to be usable for validating the kube-apiserver's serving certificate, and should not be used for any other purpose. This paragraph (and this entire doc) is misleading on that point.

It is useful to describe how to request client and serving certificates from the CSR API, but the instructions should *explicitly* state that the associated trust bundle is not distributed via the API or logically provided to pods, and that cluster admins are currently responsible for distribution to:
* things that need to verify the client certs (typically kube-apiserver and kubelet verification of client certs with `--client-ca-file`)
* things that need to verify serving certs (typically kube-apiserver verification of kubelet serving certs with `--kubelet-certificate-authority`)

Reply:
Reworded everything to stop implying cluster CA signs certs.
PTAL